File: MyLanguageDB/src/main/java/ru/herooo/mylanguagedb/entities/Color.java
package ru.herooo.mylanguagedb.entities;

import jakarta.persistence.*;

@Entity
@Table(name="color")
public class Color {

    @Id
    @GeneratedValue
    private long id;

    @Column(name="title")
    private String title;

    @Column(name="hex_code")
    private String hexCode;

    public long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getHexCode() {
        return hexCode;
    }

    public void setHexCode(String hexCode) {
        this.hexCode = hexCode;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/dto/entity/color.js
export class ColorResponseDTO {
    id;
    title;
    hexCode;

    constructor(colorJson) {
        if (colorJson) {
            this.id = colorJson["id"];
            this.title = colorJson["title"];
            this.hexCode = colorJson["hex_code"];
        }
    }
}